web/src/App.res

```diff
@@ -69,6 +69,51 @@ module MonocleNav = {
   }
 }
 
+module Login = {
+  module Modal = {
+    @react.component
+    let make = (~store: Store.t, ~setShowLoginModal) => {
+      let loginTitle = "Set your username"
+      let (_, dispatch) = store
+      let (username, setUsername) = React.useState(_ => "")
+      let onChange = (value, _) => setUsername(_ => value)
+      let onClick = _ => {
+        username->Login->dispatch
```

Comment:
I wonder if we could add a new endpoint to validate the login is valid to prevent typo?

Reply:
Yes that might be possible but I'd like to move that request in an RFE issue as it will complexifies a lot that PR.

Reply:
https://github.com/change-metrics/monocle/issues/770